### ISSUE TITLE
I Hope Add Postfix Completion 

### ISSUE THREAD

**Issue by zhangymPerson** [golang/vscode-go#1278]
I hope you can add this feature, 
like https://github.com/ipatalas/vscode-postfix-ts or https://github.com/yokoe/vscode-postfix-go

**Comment by hyangah** [comment]
@stamblerre @muirdm Can the completions offered in https://github.com/yokoe/vscode-postfix-go be implemented in gopls in the context of https://github.com/golang/go/issues/39507?

**Comment by muirdm** [comment]
> Can the completions offered in https://github.com/yokoe/vscode-postfix-go be implemented in gopls

They could be, some easier than others. Most of those snippets don't meet my personal bar for inclusion, but I'm not necessarily the gate keeper.

**Comment by hyangah** [comment]
For anyone who's following the upstream issue. @muirdm's cl is merged to master.
Please give it a try and provide feedback.

To install the master version of gopls, run the following command from an empty directory:
  `GO111MODULE=on go get golang.org/x/tools/gopls@master golang.org/x/tools@master`
 
 And with the following in the settings.json.
 
```
"gopls": {
   "ui.completion.experimentalPostfixCompletion": true
}
```

**Comment by inliquid** [comment]
@hyangah I tried both 
```
"experimentalPostfixCompletions": true,
```
and
```
"ui.completion.experimentalPostfixCompletion": true
```
following recent `gopls/v0.6.10` release notes.

And it didn't work:
![изображение](https://user-images.githubusercontent.com/6797881/114704080-d4d94f80-9d2e-11eb-99dd-ea3ac59f2d8e.png)

I'm using latest `go` with `gopls@master`.

**Comment by hyangah** [comment]
@inliquid It's plural. ui.completion.experimentalPostfixCompletion**s**

**Comment by inliquid** [comment]
@hyangah yep, sorry, that was unintended mistake when copying/pasting and so that the message start appearing which I thought I didn't notice for some reason.

Edited. So the problem is actually that it doesn't work. At least I don't see what was in release notes as an example:
![изображение](https://user-images.githubusercontent.com/6797881/114720571-cc8a1000-9d40-11eb-979c-62061653cbb0.png)

**Comment by hyangah** [comment]
@inliquid If you have gopls trace enabled, can you capture the trace and share it in a separate issue? 
Since I couldn't reproduce (see the below screenshot), I suspect another issue involved and we need a separate investigation.

![Screen Shot 2021-04-14 at 1 08 45 PM](https://user-images.githubusercontent.com/4999471/114750962-90e04d80-9d22-11eb-92e0-f80775a75dbe.png)
